docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -318,6 +329,47 @@ The ownership relationship is tracked through an `[image]` [shortcode](/develope
 which is automatically transformed into an `<img>` tag at render time. In addition to storing the image path,
 the shortcode references the database identifier of the `Image` object.
 
+### Changesets, a.k.a "Campaigns"
+
+Changes to many DataObjects can grouped together using the `ChangeSet` object, better known by its frontend name, "Campaign" (provided the `campaign-admin` module is installed). By grouping a series of content changes together as one cohesive unit, content editors can bulk publish or revert an entire body of content all at once, which affords them much more power and control over interdependent content types.
```

Comment:
Please add an `[api:SilverStripe\Versioning\ChangeSet]` link here to the class in question in place of `ChangeSet`